sys/net/gnrc/Makefile.dep

```diff
@@ -35,6 +35,9 @@ endif
 ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
   # enable default network devices on the platform
   USEMODULE += netdev_default
+endif
+
+ifneq (,$(filter netdev_default,$(USEMODULE)))
   USEMODULE += netdev
   USEMODULE += gnrc_netif
 endif
```

Comment:
At this point, what is the difference between `gnrc_netdev_default` and `netdev_default`?

Reply:
Mhhh `netdev_default` (at least in theory) should also be used to pull in default network devices for stacks other than GNRC. So this is the difference. Since this here is the GNRC dependency file, this is still observed.

Reply:
I think the goal of this PR is to make `gnrc_netdev_default` unnecessary so it can be deprecated ;-).